### ISSUE TITLE
_ character included in password field for nc-admin and nc-passwd

### ISSUE THREAD

**Issue by sunjam** [nextcloud/nextcloudpi#1131]
Confusing from `ncp-config` menu because `_` is actually there and appended to whatever the admin types.  If it is not deleted, it will be treated as part of either password when reset from this menu, which is confusing.
![Screen Shot 2020-05-04 at 11 48 54 AM](https://user-images.githubusercontent.com/1787238/81002099-8cff3800-8dfd-11ea-8b44-4c575984c732.png)

This menu item also claims to allow one to view the current password, but that is not possible afaik.

**Comment by ovpc** [comment]
Hi Sunjam
That is what it does for me
I can view and change the password.
Not sure how yours can only show "_" The first time nc-init runs, passwords for nc-admin (NC) and nc-passwd (NCP-web:4443) are generated.
![NCP-config_nc-admin-Screenshot2020-05-05_06-57-46](https://user-images.githubusercontent.com/8775469/81036102-15351a00-8e9e-11ea-98fc-171c16901252.png)

**Comment by nachoparker** [comment]
Yes, that's not an ideal solution, I kind of hacked it together real fast, I'll try to look again for a better way, but don't have much time these days..

Thanks

**Comment by nachoparker** [comment]
> This menu item also claims to allow one to view the current password, but that is not possible afaik.

we don't store passwords anymore for security purposes

**Comment by sunjam** [comment]
The only issue is the password fields have "_" in them when editing from `ncp-config`.  If the password fields were empty there would be no issue. ❤️

**Comment by nachoparker** [comment]
yeah that was introduced because empty values dont work with the TUI library as a quick cheap bad workaround

**Comment by sunjam** [comment]
> yeah that was introduced because empty values dont work with the TUI library as a quick cheap bad workaround

Cool, in that case I'd just recommend `REPLACE`
